### ISSUE TITLE
Improve `PriceRange` Accessibility

### ISSUE THREAD

**Issue by hahnwhitney** [vtex/faststore#1937]
**Background**
I've been working on a Jest/RTL accessibility test suite for FastStore's UI components. This issue isn't a bug in the sense that the component is "broken" -- it just doesn't meet the WCAG standards.

**Describe the bug**
The [`aria-label` prop on line 19](https://github.com/vtex/faststore/blob/415e43462b15a63e3c50d306e6a313ac920bd038/packages/components/src/organisms/PriceRange/PriceRange.tsx#L19) of `PriceRangeProps` is currently optional, making it possible for the `<PriceRange>` component to be inaccessible. 

**Possible Solution - 1**
The `aria-label` prop should be required; this will prevent engineers from accidentally forgetting to include an aria-label. We might also want to include an aria-label (or the option to pass an aria-label) to the [`input` field on line 144](https://github.com/vtex/faststore/blob/415e43462b15a63e3c50d306e6a313ac920bd038/packages/components/src/atoms/Slider/Slider.tsx#L144) of the `<Slider>` component.

**Comment by renatamottam** [comment]
Hi @hahnwhitney, thanks for reporting this issue in such detail, it's really helpful ✨
We created an internal task and will fix it soon.

**Comment by renatamottam** [comment]
We've lauched the FS core package version `2.1.80` fixing the bug ✨
